src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
+    private static final String DATE_PATTERN = "dd.MM.yyyy";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        SimpleDateFormat sdf = new SimpleDateFormat(DATE_PATTERN);
+        StringBuilder staffSalaryInformation = new StringBuilder("Report for period "
+                                             + dateFrom + " - " + dateTo);
+        for (String name : names) {
+            int totalSalary = 0;
+            staffSalaryInformation.append("\r\n");
```

Comment:
Use System.lineSeparator() instead of \r\n to ensure the newline character is consistent across different operating systems.